components/form/form.css

```diff
@@ -0,0 +1,124 @@
+
+
+
+@import url('../../components/framer.css');
+
+:root{
+    --form-input-border:#ced4da;
+    --form-input-border-shadow:#A5B4FC;
+    --form-input-disable:#E5E7EB;
+    --form-input-error:#E03546;
+    --form-input-error-shadow:#e6b4ba;
+    --input-txt-col:#212529;
+    --form-input-success:#03703C;
+    --form-input-success-shadow:#2FDD92;
+    --slider-box-shadow:rgba(0,0,0,0.2);
+    --form-input-warning:#ce972a;
+    --form-input-warning-shadow:#FFCC1D;
+}
+
+.label-text-primary{
+    color: var(--col-primary);
+}
+.label-text{
+    font-weight: 700;
+}
+
+.label-text-success{
+    color: var(--col-success);
+
+}
+.label-text-danger{
+    color: var(--col-danger);
+}
+
+.label-text-warning{
+    color: var(--col-warning);
+}
+.input{
+    width: auto;
+    height: 2.813rem;
```

Comment:
here you can direclty give 2.8 or 2.9 or to more micro manage you can use pixels

Reply:
> here you can direclty give 2.8 or 2.9 or to more micro manage you can use pixels

In my case these precision values are important this might break on a small device.